roles/otel_collector/handlers/main.yml

```
@@ -0,0 +1,5 @@
+- name: restart opentelemetry collector
```

```suggestion
- name: Restart OpenTelemetry Collector
```
see also 
https://opentelemetry.io/docs/contributing/style-guide/